packages/cozy-sharing/src/components/useFetchDocumentPath.jsx

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react'
+import { getFilesPaths } from '../helpers/files'
+
+export const useFetchDocumentPath = (client, document) => {
+  const [documentPath, setDocumentPath] = useState(
+    document.path ? document.path : null
+  )
+  useEffect(() => {
+    ;(async () => {
+      try {
+        const path = await getFilesPaths(client, document._type, [document])
```

Comment:
Do we still need to do the fetch if `document.path` already exists?

Reply:
I don't know there is no previous test telling us what is the expected behavior.

Since getFilesPath (which should be renamed to fetchFilesPaths...) makes a request, I think that the fetch can be useful to have the latest information.